content/deployment/install.md

```diff
@@ -209,304 +147,85 @@ Once you downloaded and verified the release, you are ready to continue with the
-                # Check if the directory structure exists
-                if not os.path.exists(cwd):
-                    os.makedirs(cwd)
-                if os.path.exists(dst_file):
-                    os.remove(dst_file)
-                    logger.debug("Removed existing " + dst_file + ".")
-                try:
-                    shutil.move(src_file, cwd)
-                except shutil.Error as exc:
-                    print(exc)
-                logger.debug("Moved " + file_ + " to " + cwd + ".")
-    logger.debug("Copying completed successfully")
-    shutil.rmtree(rand_fold)
-    logger.debug("Deleted the temporary folder.")
-
-def main():
-    """Initiate logging and run subroutines"""
-    logging.basicConfig(level=logging.INFO)
-    logger = logging.getLogger(os.path.basename(__file__))
-    args = check_arguments(logger)
-    download_rules(args, logger)
-
-if __name__ == "__main__":
-    main()
-```
-
-## Problems with installation
-
-### Apache Line Continuation
-
-In Apache 2.4.x before 2.4.11 there is a bug where the use of line
-continuations in a config size may cause the line continuation to be
-truncated. This will lead to an error similar to the following:
-
-```bash
-Syntax error on line 24 of /etc/httpd/modsecurity.d/activated_rules/RESPONSE-50-DATA-LEAKAGES-PHP.conf:
-Error parsing actions: Unknown action: \
-```
-
-This is not an error with ModSecurity or OWASP CRS. In order to fix this
-issue you can simply add a space before the continuation on the
-offending line. For more information see [apache bugzilla](https://bz.apache.org/bugzilla/show_bug.cgi?id=55910).
-
-### Anomaly Mode Doesn't Work
-
-Sometimes on IIS or Nginx users run into an instance where anamoly mode
-doesn't work as expected. In fact upon careful inspection of logs one
-would notice that rules don't fire in the order we would expect. In
-general this is a result of using the `'*'` operator within these
-environments as it does not act the same way as in Apache. In general
-within both Apache and IIS one should expliticly include the various
-files present within the OWASP CRS instead of using the `'*'`.
-
-### Webserver returns error after CRS install
-
-This is likley due to a rule triggering. For instance in some cases a
-rule is enabled that prohibits access via an IP address. Depending on
-your
-[SecDefaultAction](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#SecDefaultAction)
-and
-[SecRuleEngine](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#SecRuleEngine)
-configurations, this may result in a redirect loop or a status code. If
-this is the problem you are experiencing you should consult your
-error.log (or event viewer for IIS). From this location you can
-determine the offending rule and add an exception if neccessary see
-[false positives and tuning]({{< ref "../configuring/false_positives_tuning.md" >}}).
+include coreruleset-{{< param crs_latest_release >}}/crs-setup.conf
```

Comment:
```suggestion
include coreruleset-{{< param crs_latest_release >}}/crs-setup.conf
include coreruleset-{{< param crs_latest_release >}}/plugins/*-config.conf
include coreruleset-{{< param crs_latest_release >}}/plugins/*-before.conf
```

Reply:
For v4. I assumed this was for v4.

Reply:
Let me think how to add pages for each version. For now let's use this for 3.3.2